pipeline/outputs/elasticsearch.md

```diff
@@ -8,54 +8,59 @@ The **es** output plugin, allows to ingest your records into an [Elasticsearch](
+| :--- | :--- | :--- | :--- |
+| Host | IP address or hostname of the target Elasticsearch instance | 127.0.0.1 | Yes, default value is not applicable for NODE section of Upstream configuration, which **requires** host to be specified |
+| Port | TCP port of the target Elasticsearch instance | 9200 | Yes, default value is not applicable for NODE section of Upstream configuration, which **requires** port to be specified |
+| Path | Elasticsearch accepts new data on HTTP query path "/\_bulk". But it is also possible to serve Elasticsearch behind a reverse proxy on a subpath. This option defines such path on the fluent-bit side. It simply adds a path prefix in the indexing HTTP POST URI. | Empty string | Yes |
+| compress | Set payload compression mechanism. Option available is 'gzip' | | Yes |
+| Buffer\_Size | Specify the buffer size used to read the response from the Elasticsearch HTTP service. This option is useful for debugging purposes where is required to read full responses, note that response size grows depending of the number of records inserted. To set an _unlimited_ amount of memory set this value to **False**, otherwise the value must be according to the [Unit Size](../../administration/configuring-fluent-bit/unit-sizes.md) specification. | 4KB | Yes |
+| Pipeline | Newer versions of Elasticsearch allows to setup filters called pipelines. This option allows to define which pipeline the database should use. For performance reasons is strongly suggested to do parsing and filtering on Fluent Bit side, avoid pipelines. |  | Yes |
+| AWS\_Auth | Enable AWS Sigv4 Authentication for Amazon OpenSearch Service | Off | Yes |
+| AWS\_Region | Specify the AWS region for Amazon OpenSearch Service |  | Yes |
+| AWS\_STS\_Endpoint | Specify the custom sts endpoint to be used with STS API for Amazon OpenSearch Service |  | Yes |
+| AWS\_Role\_ARN | AWS IAM Role to assume to put records to your Amazon cluster |  | Yes |
+| AWS\_External\_ID | External ID for the AWS IAM Role specified with `aws_role_arn` |  | Yes |
+| AWS\_Service\_Name | Service name to be used in AWS Sigv4 signature. For integration with Amazon OpenSearch Serverless, set to `aoss`. See the [FAQ](opensearch.md#faq) section on Amazon OpenSearch Serverless for more information. | es | Yes |
+| AWS\_Profile | AWS profile name | default | Yes |
+| Cloud\_ID | If you are using Elastic's Elasticsearch Service you can specify the cloud\_id of the cluster running. The Cloud ID string has the format `<deployment_name>:<base64_info>`. Once decoded, the `base64_info` string has the format `<deployment_region>$<elasticsearch_hostname>$<kibana_hostname>`. |  | No |
+| Cloud\_Auth | Specify the credentials to use to connect to Elastic's Elasticsearch Service running on Elastic Cloud |  | Yes |
+| HTTP\_User | Optional username credential for Elastic X-Pack access |  | Yes |
+| HTTP\_Passwd | Password for user defined in HTTP\_User |  | Yes |
+| Index | Index name | fluent-bit | Yes |
```

Comment:
```suggestion
| AWS\_STS\_Endpoint | Specify the custom sts endpoint to be used with STS API for Amazon OpenSearch Service. | _none_ | Yes |
| AWS\_Role\_ARN | AWS IAM Role to assume to put records to your Amazon cluster. | _none_ | Yes |
| AWS\_External\_ID | External ID for the AWS IAM Role specified with `aws_role_arn`. | _none_ | Yes |
| AWS\_Service\_Name | Service name to be used in AWS Sigv4 signature. For integration with Amazon OpenSearch Serverless, set to `aoss`. See the [FAQ](opensearch.md#faq) section on Amazon OpenSearch Serverless for more information. | `es` | Yes |
| AWS\_Profile | AWS profile name. | `default` | Yes |
| Cloud\_ID | If you are using Elastic's Elasticsearch Service you can specify the `cloud_id` of the cluster running. The Cloud ID string has the format `<deployment_name>:<base64_info>`. After decoding, the `base64_info` string has the format `<deployment_region>$<elasticsearch_hostname>$<kibana_hostname>`. | _none_ | No |
| Cloud\_Auth | Specify the credentials to use to connect to Elastic's Elasticsearch Service running on Elastic Cloud. | _none_ | Yes |
| HTTP\_User | Optional username credential for Elastic X-Pack access. | _none_ | Yes |
| HTTP\_Passwd | Password for user defined in HTTP\_User. | _none_ | Yes |
| Index | Index name. | `fluent-bit` | Yes |
```

Reply:
Hi @cnorris-cs,

I'm sorry, but it doesn't look like other parts of Fluent Bit documentation (like https://github.com/fluent/fluent-bit-docs/blob/master/pipeline/outputs/forward.md) wrap default values with backticks, i.e. the parts of your suggestion like replacing `fluent-bit` with 

```md
`fluent-bit`
```

seem to not match existing documentation style.

Same for:

```md
_none_
```

in "Default" column for the keys without default value - the most of existing documentation keeps such cells empty.

Please note also that values in "Description" and "Default" columns were mostly copied from original version of this file, so I'm not sure if this pull request is allowed to change these values (because this pull request is about overriding only).

Thank you.

Reply:
We're actually going to be working on updating the documentation to be along the lines of what I suggested. Empty cells don't always communicate the concept of "nothing". Values to be entered should be consistent, and marking them as code help to signify that they're items to be entered.

So, we're making these suggestions to help us get where we want to be with the docs, knowing that it'll take some time to get there. But moving in that direction sooner than later helps with all of this.

(And most of the changes to the description column were to do things like add punctuation and to incorporate grammar changes. I do know of one line that I did some work on that would need review -- the ElasticSearch line -- but it would benefit from some changes for clarity.)